app/frontend/src/internationalization/index.js

```diff
@@ -45,7 +45,7 @@ const messages = {
 
 // Create VueI18n instance with options
 export default new VueI18n({
-  locale: 'en', // set locale
+  locale: sessionStorage.getItem('lang') !== null ? sessionStorage.lang : 'en', // set locale
```

Comment:
This can just be `sessionStorage.getItem('lang')` since the fallbackLocale is already `en`.